File: apps/flights-service-admin/src/api/airline/AirlineUpdateInput.ts
import { FlightWhereUniqueInput } from "../flight/FlightWhereUniqueInput";
import { FlightUpdateManyWithoutAirlinesInput } from "./FlightUpdateManyWithoutAirlinesInput";

export type AirlineUpdateInput = {
  arrivalFlights?: FlightWhereUniqueInput | null;
  country?: string | null;
  flights?: FlightUpdateManyWithoutAirlinesInput;
  name?: string | null;
};

File: apps/flight-catalog/src/APIs/Airline/Dtos/AirlineUpdateInput.cs
namespace FlightCatalog.APIs.Dtos;

public class AirlineUpdateInput
{
    public string? Country { get; set; }

    public DateTime? CreatedAt { get; set; }

    public List<string>? Flights { get; set; }

    public string? Id { get; set; }

    public string? Name { get; set; }

    public DateTime? UpdatedAt { get; set; }
}
